harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/AbstractHarvestError.java

```diff
@@ -0,0 +1,24 @@
+package org.fao.geonet.kernel.harvest.harvester;
+
+import org.fao.geonet.monitor.harvest.AbstractHarvesterErrorCounter;
+
+import com.yammer.metrics.core.Counter;
+
+import jeeves.monitor.MonitorManager;
+import jeeves.server.context.ServiceContext;
+
+public class AbstractHarvestError {
+	
+	public AbstractHarvestError(ServiceContext context) {
+		super();
+		// we don't catch anything because this is not critical
+		// if the following code is not executed, we just want to
+		// avoid to raise an exception.
+		try {
+			MonitorManager mm = context.getBean(MonitorManager.class);
```

Comment:
> it might be interesting to reconsider the metrics introduction into GN completely.

Maybe ! My main concern was that we also have new type of harvester (eg. the WFS feature one) which does not rely on Jeeves at all. So I'm wondering how we could do it better on the long run.


BTW, +1 for merging if you need that metric.